src/imap/mod.rs

```diff
@@ -1449,14 +1449,24 @@ async fn prefetch_is_reply_to_chat_message(
     false
 }
 
-async fn prefetch_should_download(
+pub(crate) async fn prefetch_should_download(
     context: &Context,
     headers: &[mailparse::MailHeader<'_>],
     show_emails: ShowEmails,
 ) -> Result<bool> {
     let is_chat_message = headers.get_header_value(HeaderDef::ChatVersion).is_some();
     let is_reply_to_chat_message = prefetch_is_reply_to_chat_message(context, &headers).await;
 
+    let maybe_ndn = if let Some(subject) = headers.get_header_value(HeaderDef::Subject) {
+        subject.to_ascii_lowercase().contains("fail")
+    } else {
+        false
+    } || if let Some(ctype) = headers.get_header_value(HeaderDef::ContentType) {
+        ctype.starts_with("multipart/report")
```

Comment:
this condition is always false as the `ContentType` is not prefetched - to save traffic, we only prefetch some selected headers defined in `PREFETCH_FLAGS`.

maybe it better to just to check if `From:` contains the string `deamon`, at least all examples would work and also heuristics would work better as this is the same condition as in `heuristically_parse_ndn()`.

alternatively, of course, we could add `ContentType` to `PREFETCH_FLAGS`, however, the From-check seems to be required anyway and so this check seems to be a bit redundant (at the cost of more traffic on prefetch)

Reply:
Done.